src/Components/forms/inputs/EmployerDropDown.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react'
+import { Label } from 'reactstrap'
+import Select from 'react-select'
+
+const EmployerDropDown = ({ onChange, isEmpty, arrayList, name }) => {
+  const [value, setValue] = useState('')
+  const employerSorter = (a, b) => {
+    const aIndex = a.value.toLowerCase().indexOf(value.toLowerCase())
+    const bIndex = b.value.toLowerCase().indexOf(value.toLowerCase())
+    return a.value === 'Other' || b.value === 'Other'
+      ? 1
+      : aIndex > bIndex
+      ? 1
+      : aIndex < bIndex
+      ? -1
+      : 0
+  }
+  const employersList = arrayList
+    .map(item => ({
+      value: item.name,
+      label: item.name
+    }))
```

Comment:
The `item` has _two_ properties, `name` and `_id`. Yes, for all the values we have they're the same (and yet it seems nobody thought to `map` over a list of strings 🤷), but is that accidental or essential duplication? I would guess the point of that was that we wouldn't _necessarily_ have to have the visible name be the same as the stored ID, but here you ignore the latter entirely. If someone later adds an employer where the name and ID _do_ differ, which the array in `data.json` has an affordance for, this component will send the wrong value to the API. I think it should be:

```suggestion
    .map(item => ({
      value: item._id,
      label: item.name
    }))
```
or use [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#syntax):
```suggestion
    .map(({ _id, name }) => ({ value: _id, label: name }))
```

Reply:
changed it to use _id  and name values